tgs/src/main/cpp/tgs/DelaunayTriangulation/DelaunayTriangulation.cpp

```diff
@@ -611,7 +611,14 @@ Edge EdgeIterator::operator*()
   return _e;
 }
 
-void EdgeIterator::operator++(int)
+Edge EdgeIterator::operator++(int)
+{
+  Edge tmp(_e);
+  operator++();
+  return tmp;
+}
+
+Edge& EdgeIterator::operator++()
```

Comment:
Created pre and postfix increment operators so that the prefix (more efficient) could be used.